03/index.html

```diff
@@ -6,9 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #03</title>
     <link rel="stylesheet" href="./styles/global.css">
-    <!--<link rel="stylesheet" href="./styles/mobile.css">-->
-    <!--<link rel="stylesheet" href="./styles/tablet.css">-->
-    <link rel="stylesheet" href="./styles/desktop.css">
+    <link rel="stylesheet" media="(min-width: 599px) and (max-width:1299px)" href="./styles/mobile.css">
```

Comment:
Jeżeli definiujesz `mobile` i w zadaniu piszę że jest to przedział do `599px` to tutaj powinieneś wpisać: `(max-width: 599px)`.

PS. Plik ten nie zawiera żadnego kodu więc obecnie to bez znaczenia, ale musisz uważać na odpowiednie przedziały.